mappings/net/minecraft/util/math/random/Random.mapping

```diff
@@ -54,6 +54,6 @@ CLASS net/minecraft/class_5819 net/minecraft/util/math/random/Random
 	METHOD method_43057 nextFloat ()F
 	METHOD method_43058 nextDouble ()D
 	METHOD method_43059 nextGaussian ()D
-	METHOD method_43385 nextPredictable (DD)D
+	METHOD method_43385 nextTriangular (DD)D
 		ARG 1 base
 		ARG 3 variance
```

Comment:
This isn't a variance, but more like a "radius", like the difference from the mean (should rename "base" too) to the upper and lower limit.

Reply:
Would `deviation` work?

```suggestion
		ARG 3 deviation
```

Reply:
> Would `deviation` work?

Sure, as long as it doesn't overlap with statistical terms.

Reply:
These are the typical statistical terms when describing a triangular distribution:

![triangular-distribution](https://user-images.githubusercontent.com/24855774/169601704-8440e229-3fe8-443e-a634-4e305f9b0370.png)

The parameters `mean` and `deviation` would align with these terms.